ui/packages/security-idp-server/src/main/webapp/js/view/Login.view.js

```diff
@@ -76,6 +84,13 @@ define([
         },
         error: function() {
           $('#loginerror' + view.model.get('AuthMethod')).show()
+          $('#loginspinner').hide()
+          $('#username').attr('readonly', false)
+          $('#password').attr('readonly', false)
+          $('.btn-upsignin').attr('disabled', false)
+          $('.btn-upclear').attr('disabled', false)
+          $('.btn-pkisignin').attr('disabled', false)
+          $('.btn-guestsignin').attr('disabled', false)
```

Comment:
✏️ May improve readability if these frontend element changes were extracted into their own method with a descriptive name. For example:
```
    hideSpinnerAndEnableInput: function() {
          $('#loginspinner').hide()
          $('#username').attr('readonly', false)
          $('#password').attr('readonly', false)
          $('.btn-upsignin').attr('disabled', false)
          $('.btn-upclear').attr('disabled', false)
          $('.btn-pkisignin').attr('disabled', false)
          $('.btn-guestsignin').attr('disabled', false)
    }
```